pkg/service/scheduler/model.go

```diff
@@ -30,6 +30,7 @@ const (
 	RepairTask         TaskType = "repair"
 	SuspendTask        TaskType = "suspend"
 	ValidateBackupTask TaskType = "validate_backup"
+	PurgeBackupsTask   TaskType = "purge_backups"
```

Comment:
Leftover?

Reply:
Yes, my mistake. Removed this and other mentions.